stdlib/asyncio/base_events.pyi

```diff
@@ -11,7 +11,11 @@ from socket import AddressFamily, SocketKind, _Address, _RetAddress, socket
 from typing import IO, Any, Awaitable, Callable, Coroutine, Generator, Sequence, TypeVar, overload
 from typing_extensions import Literal
 
-if sys.version_info >= (3, 7):
+if sys.version_info >= (3, 10):
+    from contextvars import Context
```

Comment:
Optional: We could also move the import into its own block. I found it a bit weird that they were combined before.